Resources/Documentation/hardware.md

```diff
@@ -1,14 +1,14 @@
 # Needed Hardware
 
-- a computer running a recent macOS (vrecord 2017-10-17 has been tested on macOS 10.10, 10.11, 10.12 and 10.13)
-- a Blackmagic card from:
+- a computer running a recent version of macOS or select linux distributions (vrecord 2019-01-25 has been tested on macOS 10.11, 10.12, 10.13, and 10.14, and Ubuntu 18.04)
```

Comment:
Sorry for one more tiny thing! It has also been tested on Ubuntu 16.04!

Reply:
added!